packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -223,7 +223,7 @@ class _NetworkScreenBodyState extends State<NetworkScreenBody>
   void _showFilterDialog() {
     showDialog(
       context: context,
-      builder: (context) => FilterDialog(
+      builder: (context) => FilterDialog<NetworkController, NetworkRequest>(
```

Comment:
why aren't these types able to be inferred correctly?

Reply:
we need to pass these types along to the Filter classes and Filter mixin that expect a generic type to be defined. This lets us use the same filter code across different screens with different data types.